contributing/code/core_team.rst

```diff
@@ -27,15 +27,19 @@ The Symfony Core groups, in descending order of priority, are as follows:
 * Elects members in any other group;
 * Merges pull requests in all Symfony repositories.
 
-2. **Mergers**
+2. **Mergers Team**
 
 * Merge pull requests for the component or components on which they have been
   granted privileges.
 
-3. **Deciders**
+3. **Deciders Team**
 
 * Decide to merge or reject a pull request.
 
+4. **Security Team**
+
+* Decide to merge or reject a pull request related to security issues.
```

Comment:
I think the security team is orthogonal to the other ones. Michael could be also a merger for some components for instance. It's also more than deciding things. It's about managing the security process from reporting to releasing.